README.md

```
@@ -1,42 +1,49 @@
-# HTTP-Server
+# ccerve
 
-## Introduction:
+A HTTP server written from scratch in C++.
 
+### How to run:
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)